src/main/java/DukeBot/Ui.java

```diff
@@ -0,0 +1,96 @@
+package DukeBot;
+
+import java.util.Scanner;
+
+/**
+ * Encapsulates the User Interface.
+ */
+public class Ui {
+
+    private Scanner scanner;
+
+    /**
+     * Initialises the Ui object and a Scanner that reads the user's input.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Prints a line onto the output.
+     */
+    public void showLine() {
```

Comment:
Instead of `showLine`, do you think that renaming to `printLine` might be better naming?